deliverylocation/tests/test_deliverylocation.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from deliverylocation.models import Supplier, DeliveryLocation
+from django.core.exceptions import ValidationError
+import datetime
+
+test_data = [
+                ('taxas', 'firstname', 'lastname', 'password', 'supplier', 'business name'),
+                ('ed', 'edd', 'eddy', 'jawbreaker101', 'supplier', 'jawbreaker inc'),
+        ]
+
+
+@pytest.fixture
+def supplier0():
+    return Supplier(test_data[0])
+
+
+@pytest.fixture
+def deliverylocation0(supplier0):
+    return DeliveryLocation(user_name=supplier0, location="Qiryat Shemona", date=datetime.date(2022, 12, 30))
+
+
+@pytest.fixture
+def deliverylocation1(supplier0):
+    return DeliveryLocation(user_name=supplier0, location="Haifa", date=datetime.date(2022, 12, 31))
+
+
+class TestDeliveryLocationModel:
+    @pytest.mark.django_db()
+    def test_add_delivery_location(self, supplier0, deliverylocation0, deliverylocation1):
+        supplier0.save()
+        deliverylocation0.add_delivery_location()
```

Comment:
I think it would have make sense to make this a fixture. Or just put it inside of the `deliverylocation0` fixture.

Reply:
tried to do that but it makes it more complicated to understand what is being saved and deleted, and it fails some tests due the deletion and saving back